authHelpers/authRoutes.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
```

Comment:
Your naming convention is a bit odd. `authHelpers` and `authRoutes` don't necessarily describe what's going on. A directory for `routes` and then individual files for the `register`, `login`, `logout`, etc routes, or all of those routes in an `index` file in the `routes` directory would be more clear.